lighthouse-core/report/html/report-styles.css

```diff
@@ -39,7 +39,7 @@
   --report-text-color: var(--color-black-900);
   --report-font-size: 16px;
   --report-line-height: 24px;
-  --snippet-color: hsl(206, 6%, 25%);
+  --snippet-color: var(--color-black-800);
```

Comment:
look at `Background and foreground colors do not have a sufficient contrast ratio.` in the now deployment for a snippet.